src/sentry/relay/config/__init__.py

```diff
@@ -188,7 +188,9 @@ def _get_project_config(project, full_config=True, project_keys=None):
     # or just return uniform rule
     if allow_dynamic_sampling_basic:
         try:
-            cfg["config"]["dynamicSampling"] = {"rules": [generate_uniform_rule(project)]}
+            cfg["config"]["dynamicSampling"] = {
+                "rules": [generate_rules(project, boost_releases=True)]
```

Comment:
General question: Have you considered writing the rules into project option `sentry:dynamic_sampling` in https://github.com/getsentry/sentry/pull/40403, instead of generating them from redis here on the fly? That would simplify the code in this module, but it would also simplify building a UI that displays the currently active booster rules (because it would only have to look at one project option instead of grabbing information from multiple places).